docs/developer/contributing.rst

```diff
@@ -231,8 +232,9 @@ conda. With cmder open in the pymedphys directory, run:
 .. code:: bash
 
     conda install pymedphys --only-deps
```

Comment:
This line here will also break after we make our release. `pymedphys` will depend on our custom packages only. Maybe what we could do is bundle this into yarn so that all we need to do is run `yarn` and it handles the rest.